packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```
@@ -484,6 +484,26 @@ export default class BackupNg {
         })
       }
       return result.id
+    } catch (error) {
+      if (
```

Don't swallow other errors.